docker/Dockerfile

```diff
@@ -9,7 +9,8 @@ RUN apt-get update && apt-get install -y \
     protobuf-compiler \
     curl \
     build-essential \
-    pkg-config
+    pkg-config \
```

Comment:
We have to update the README, and the developer must install `ptyhon3-dev` on their machines.